tsconfig.json

```diff
@@ -13,14 +13,14 @@
 		"noEmit": true,
 		"paths": {
 			"@application/*": ["assets/src/application/*"],
-			"@appConstants/*": ["assets/src/application/constants/*"],
-			"@appServices/*": ["assets/src/application/services/*"],
 			"@appCalendars/*": ["assets/src/application/ui/calendars/*"],
+			"@appConfig/*": ["assets/src/application/valueObjects/config/*"],
+			"@appConstants/*": ["assets/src/application/constants/*"],
 			"@appDisplay/*": ["assets/src/application/ui/display/*"],
 			"@appForms/*": ["assets/src/application/ui/forms/*"],
 			"@appInputs/*": ["assets/src/application/ui/input/*"],
 			"@appLayout/*": ["assets/src/application/ui/layout/*"],
-			"@appConfig/*": ["assets/src/application/valueObjects/config/*"],
+			"@appServices/*": ["assets/src/application/services/*"],
```

Comment:
👍 thnx for staying on top of this

I've noticed there are a bunch of VSC extensions that can do this automagically like:

- [Sort lines](https://marketplace.visualstudio.com/items?itemName=Tyriar.sort-lines) 
- [TypeScript Import Sorter](https://marketplace.visualstudio.com/items?itemName=mike-co.import-sorter)
- [Transformer](https://marketplace.visualstudio.com/items?itemName=dakara.transformer)

do you use anything like that? if so which one?
the last two on that list looks pretty interesting

Reply:
I had some troubles with svgs being recognized initially and tried smth here, that's why it caught my attention in the first place

Reply:
I wasn't aware of those extensions and was doing ordering ... manually;
thx, will check each of them, the approach seems promising